packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -85,10 +88,86 @@ export interface ITreeConfigurationOptions {
 	 * bit slower.
 	 */
 	enableSchemaValidation?: boolean;
+
+	/**
+	 * A flag used to opt into strict rules ensuring that the schema avoids cases which can make the type of nodes ambiguous when importing or exporting data.
+	 * @defaultValue `false`.
+	 *
+	 * @remarks
+	 * When this is true, it ensures that the compile time type safety for data when constructing nodes is sufficient to ensure than the runtime behavior will not give node data ambiguity errors.
+	 *
+	 * This ensures that the canonical JSON like representation of the node of any union in the tree is lossless and unambiguous.
+	 * This canonical JSON like representation consists of arrays, plain old JavaScript objects with string keys, booleans, numbers (excluding NaN, -0 and infinities), strings, null and IFluidHandles.
+	 * Assuming any IFluidHandles get special handling, its JSON compatible, as well as compatible with the node creation APIs (such as schema class constructors).
+	 *
+	 * Currently these cases can cause ambiguity in a union:
+	 *
+	 * - More than one ArrayNode type: it's impossible to tell which array type is intended in the case of empty arrays (`[]`).
```

Comment:
Remember that a blank line is required between each list element in TSDoc comments.
https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#line-breaks